Source/Core/Core/Slippi/SlippiReplayComm.cpp

```diff
@@ -3,6 +3,7 @@
 #include "Common/CommonPaths.h"
 #include "Common/Logging/LogManager.h"
 #include "Core/ConfigManager.h"
+#include <cctype>
```

Comment:
what does this do?

Reply:
fixes an error I kept getting about std::isspace not being a member